core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpParser.java

```diff
@@ -219,13 +225,25 @@ public ParseResult parseMeasurement(long bufHi) {
                         bufAt += 1;
                         break;
                     } else if (isQuotedFieldValue) {
-                        return ParseResult.ERROR;
+                        return getError();
                     }
 
                 default:
                     appendByte = true;
                     nextValueCanBeOpenQuote = false;
                     break;
+
+                case '\0':
+                    LOG.info().$("could not parse [byte=\\0]").$();
+                    return getError();
+                case '/':
+                case '.':
```

Comment:
`.` is quite Innocent without slashes, I don't see why we should restrict it

Reply:
there is big performance hit differentiating `..` and `.` and `../`. This is compilated by TCP fragmentation, where `.` can be in one buffer and `./` in another. Such case requires penalising second pass over data or significant state machine changes

Reply:
I'd think refusing anything with `/` would cover fragmented cases of `./`, `../`

Reply:
And I don't see problems with `.`, `..` anywhere in table, column name

Reply:
agreed, changed